athf/mcp/tools/search_tools.py

```
@@ -0,0 +1,152 @@
+                    " ".join(fm.get("tactic", []) if isinstance(fm.get("tactic"), list) else [str(fm.get("tactic", ""))]),
+                    parsed.get("content", ""),
+                ]
+                corpus_texts.append(" ".join(str(p) for p in text_parts))
+                corpus_hunts.append({
+                    "hunt_id": fm.get("hunt_id", f.stem),
+                    "title": fm.get("title", ""),
+                    "technique": fm.get("technique", ""),
+                    "status": fm.get("status", ""),
+                })
+            except Exception:
+                continue
```

Ruff (0.15.6)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)